src/suppliers/model/Supplier.ts

```diff
@@ -9,6 +10,7 @@ export default interface Supplier {
   id: string;
   integrationId: string;
   name: string;
+  playSettings?: PlaySettings;
```

Comment:
@lmkerr can you add a quick assertion in the SupplierService.spec.ts. It looks like in the one test we are returning thee play settings, but would be nice to add the assertion there so removing this would break future tests.